main.tf

```diff
@@ -57,14 +59,22 @@ resource "spacelift_policy" "custom" {
   type = upper(split(".", each.key)[0])
   name = format("%s %s Policy", upper(split(".", each.key)[0]), title(replace(split(".", each.key)[1], "-", " ")))
   body = file(format("%s/%s.rego", var.policies_by_name_path, each.key))
+
+  space_id = var.attachment_space_id
 }
 
 module "stacks" {
   source = "./modules/stack"
 
   for_each = local.spacelift_stacks
 
+  space_id = coalesce(var.stacks_space_id, try(data.spacelift_current_space.administrative[0].id, null), "legacy")
+
   enabled                   = each.value.enabled
+  dedicated_space           = try(each.value.settings.spacelift.dedicated_space, false)
+  space_name                = try(each.value.settings.spacelift.space_name, null)
+  parent_space_id           = try(each.value.settings.spacelift.parent_space_id, null)
+  space_inheritance         = try(each.value.settings.spacelift.space_inheritance, false)
```

Comment:
maybe?
```suggestion
  inherit_entities         = try(each.value.settings.spacelift.space_inheritance, false)
```

Reply:
It feels a little vague to me -- I feel that it's important to explicitly mention that it's relating to "spaces".

Reply:
the issue is that then we have a "cloudpossE" word that then maps to a "spacelift" word which deviates from the spacelift docs. Wouldn't it be easier if it was the same naming convention?

Reply:
I'm not sure that I follow to be honest.. what are the "cloudposse" and "spacelift" words you're referring to exactly?

Reply:
Cloudposse yaml spacelift settings vs spacelift resource arguments

Reply:
cc @aknysh